app/jobs/scheduled/bookmark_reminder_notifications.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+module Jobs
+
+  # Runs periodically to send out bookmark reminders, capped at 300 at a time.
+  # Any leftovers will be caught in the next run, because the reminder_at column
+  # is set to NULL once a reminder has been sent.
+  class BookmarkReminderNotifications < ::Jobs::Scheduled
+    MAX_REMINDER_NOTIFICATIONS_PER_RUN = 300
+    JOB_RUN_NUMBER_KEY = 'jobs_bookmark_reminder_notifications_job_run_num'.freeze
+    AT_DESKTOP_CONSISTENCY_RUN_NUMBER = 6
```

Comment:
```suggestion
    MAX_REMINDER_NOTIFICATIONS_PER_RUN ||= 300
    JOB_RUN_NUMBER_KEY ||= 'jobs_bookmark_reminder_notifications_job_run_num'.freeze
    AT_DESKTOP_CONSISTENCY_RUN_NUMBER ||= 6
```

This removes error messages in case the class is reloaded.